front/backup.php

```diff
@@ -212,7 +216,9 @@ function get_def($DB, $table) {
  * @param $DB        DB object
  * @param $dumpFile  dump file
  * @param $duree     max delay before refresh
-**/
+ *
+ * @return bool
```

Comment:
To be consistent, use full name such as `boolean` or `integer` instead of `bool` and `int`.